stories/button/linkButton/index.stories.tsx

```diff
@@ -89,6 +88,10 @@ export const WithControls = () => (
   </LinkButton.Container>
 );
 
+WithControls.parameters = {
+  chromatic: { disable: true },
```

Comment:
does disablement of the controls add on serve the purpose of making the snapshots cleaner, in that there are no additional UI elements on screen that could interfere with the chromatic checks?

Reply:
parameters that include `{ chromatic: { disable: true } }` means we do not take a snapshot of it in Chromatic: https://www.chromatic.com/docs/ignoring-elements#ignore-stories

Essentially--with this LinkButton code as an example--the [LinkButton story](https://radiance-ui.curology.com/?path=/story/components-button-linkbutton--with-controls) is entirely duplicative of the default primary button captured in the [Default story](https://radiance-ui.curology.com/?path=/story/components-button-linkbutton--default). Since visual changes will require manual QA of the diff, I consider stories that are created solely to use add-ons (whether Knobs or Controls, which is what Storybook recommends going forward) not appropriate candidates for the visual regression testing, since they're pretty much just overhead without any precise visual distinction from the other stories. Since we had them before, though, I wanted to keep them in Storybook, just not include them in Chromatic.

Let me know if this doesn't answer your question! I still need to update the documentation in Notion once this is merged in, but I wanted to make sure it was ready to show (even if not *fully* complete, i.e. documentation is updated) in Demos and Discussions.

Reply:
Yup makes total sense - thank you